src/hotspot/share/gc/g1/g1BlockOffsetTable.cpp

```diff
@@ -309,7 +308,6 @@ void G1BlockOffsetTablePart::alloc_block_work(HeapWord** threshold_, size_t* ind
 
   // index_ and threshold_ updated here.
```

Comment:
Stale comment about `_index`. Maybe just "Finally update threshold_." at the end, but simply removing the comment seems as fine.